source/aks/rebuilding-aks-clusters.html.md.erb

```diff
@@ -43,7 +43,49 @@ Click on **Run** to start the pipeline.
 
 #### After deployment of a cluster
 
-- Update Civil DNS entries to point to the active Jenkins cluster (until they switch to using platform-hmcts-net). [Update active jenkins-cluster IP value in DNS](https://github.com/hmcts/azure-private-dns/pull/428/files) and [update existing platform-hmcts-net entries](https://github.com/hmcts/azure-private-dns/pull/430/files).
+- Change Jenkins to use the other cluster, e.g. [cnp-flux-config#19886](https://github.com/hmcts/cnp-flux-config/pull/19886/files) .
+  - If not updated, you can change manually providing PR has been approved and merged [View Jenkins Configuration](https://build.platform.hmcts.net/configure)
```

Comment:
Maybe change wording to review in Jenkins mandatory and update if needed?

Reply:
sure make sense